Extension/src/LanguageServer/ui.ts

```diff
@@ -294,6 +307,26 @@ export class OldUI implements UI {
         }
     }
 
+    private compilerTimout?: NodeJS.Timeout;
+    public async showCompilerStatusIcon(show: boolean): Promise<void> {
+        if (!telemetry.showStatusBarIntelliSenseIndicator()) {
+            return;
```

Comment:
Does `this.compilerTimout` need to be reset for this case? Or will it be undefined already when `!telemetry.showStatusBarIntelliSenseIndicator()`?

Reply:
No, we should never enter the rest of the function (the result of this function should never change during the session), so the timeout should never be set.

Reply:
If it reaches this case then I believe it will be undefined already